### ISSUE TITLE
Video and Audio not working

### ISSUE THREAD

**Issue by HemangiVekaria** [BelledonneCommunications/linphone-android#1611]
**Describe the bug**
When doing audio calling one device to the second device cannot hear a voice in any device. Also when requesting for video at that time can see only your own video not other user's videos. I cloned code from the master today and this issue I faced. With previous versions have not checked what happens. Also, I downloaded the app from Playstore https://play.google.com/store/apps/details?id=org.linphone and in that also a video is not working. The issue is with a specific device or what? Can anyone help me?

I am trying to use Linphone SDK in my app and tried to make audio and video work since last 4 days but got no success.

**Comment by Viish** [comment]
Can you attach logs showing the issue please? It's probably a network problem.

**Comment by HemangiVekaria** [comment]
![image](https://user-images.githubusercontent.com/17717369/153595451-08a32cfe-a60b-4cf1-b8ad-e84f5dd98eb5.png)
I can see screen like this when accepting video request.

**Comment by Viish** [comment]
That's not relevant. Please attach SDK logs.

**Comment by HemangiVekaria** [comment]
https://we.tl/t-gSKWGmURYv  May this will help.

**Comment by Viish** [comment]
Try enabling STUN & ICE in your account settings (you can use stun.linphone.org STUN server if you don't have one).

**Comment by HemangiVekaria** [comment]
We are using our own flexisip server. With the same server in iOS with linphone sdk audio and video both working but in android it's an issue.

**Comment by Viish** [comment]
Have you tried on a real device instead of an Android emulator?

**Comment by HemangiVekaria** [comment]
yes tried in real device also... same issue there..

**Comment by Viish** [comment]
And did you tried with ICE / STUN? If you have a flexisip instance it can act as a STUN server itself.

**Comment by HemangiVekaria** [comment]
Yes i have flexisip instance so i think i there's nothing to change on that server. IS there any setting that i have to do in linphone-andorid app setting? i already added STUN there

**Comment by Viish** [comment]
You also have to enable ICE toggle setting.

**Comment by HemangiVekaria** [comment]
Have also enabled that. Audio is working fine with the latest SDK version. Issues is with Video. It's not working in this app and also not working in app which i currently developed. If there's any issue with our own flexisip server then in that case video call should not work in iOS app. But somehow video calling is working in iOS without any issue

**Comment by Viish** [comment]
I see from your screenshot you are using master branch of linphone-android (and thus master branch of linphone-sdk).
Do you reproduce the issue with 4.6.1 linphone-android release with 5.1.0 SDK ?

**Comment by HemangiVekaria** [comment]
Yes tried with latest code from master branch and still facing same issue

**Comment by Viish** [comment]
Please attach logs from 4.6.1 android release showing the issue.
